### PR TITLE
Explain when you can use unique constraints

### LINKED ISSUE #131: Add question about unique constraints to faq
This is from our slack room:

> ERROR:  cannot distribute relation: "foo"
> DETAIL:  Distributed relations cannot have UNIQUE constraints or PRIMARY KEYs that do not include the partition column. 
> what does this error mean?

@craigkerstiens says it's a common question, so add it to the FAQ.

### PR DESCRIPTION
**Pull request by begriffs**
Closes #131

### CODE REVIEW

File: faq/faq.rst [hunk 1]
```diff
@@ -72,3 +72,10 @@ How does citus support count(distinct) queries?
 -----------------------------------------------
 
 Citus can push down count(distinct) entirely down to the worker nodes in certain situations (for example if the distinct is on the distribution column or is grouped by the distribution column in hash-partitioned tables). In other situations, Citus uses the HyperLogLog extension to compute approximate distincts. You can read more details on how to enable approximate :ref:`count_distinct`.
+
+In which situations may I add a uniqueness constraint to a distributed table?
```
Comment (sumedhpathak) [inline faq/faq.rst]: Looks good to me. However, @samay-sharma, @saicitus Any more content you'd want to add or re-word?

File: faq/faq.rst [hunk 2]
```diff
@@ -72,3 +72,10 @@ How does citus support count(distinct) queries?
...
+-----------------------------------------------------------------------------
+
+Citus is able to enforce a primary key or uniqueness constraint on columns only when one of the columns is the distribution column. In particular this means that if a single column constitutes the primary key then it has to be the distribution key as well.
```
Comment (samay-sharma) [inline faq/faq.rst]: Minor, slight re-phrasing. Feel free to incorporate or ignore.

"Citus is able to enforce a primary key or uniqueness constraint only when the primary key constraint contains the distribution column. In particular this means that if a single column constitutes the primary key then it has to be the distribution key as well."

File: faq/faq.rst (hunk 1 above)
Comment (samay-sharma) [inline faq/faq.rst]: Maybe rephrase to "In which situations are uniqueness constraints supported on distributed tables?"

**Comment by samay-sharma** [comment]
@begriffs : Looks good to me. Once you evaluate the 2 minor comments, feel free to :shipit:
